admin/class-yoast-form.php

```diff
@@ -298,8 +298,8 @@ public function light_switch( $var, $label, $buttons = array(), $reverse = true,
 			$this->options[ $var ] = 'on';
 		}
 
-		$class           = 'switch-light switch-candy switch-yoast-seo';
-		$aria_labelledby = esc_attr( $var ) . '-label';
+		$class = 'switch-light switch-candy switch-yoast-seo';
+		$id    = esc_attr( $var ) . '-label';
```

Comment:
`esc_attr` should be applied when echo'ing the string, not here.

When applying `esc_attr` on a string, the whole string should be included, so `esc_attr( $var . '-label' );` would be the correct usage.

Reply:
OK will move it. Now that I think at it a bit better, `$var` is not an user input, nor it gets translated. Its value is something that we hardcode and it's under our control. Escaping seems a bit unnecessary, but will do as a best practice 👍